dev/com.ibm.ws.jca_fat_duplicates/fat/src/com/ibm/ws/jca/fat/duplicates/DuplicateResourceAdaptersTest.java

```diff
@@ -55,6 +56,7 @@ public static void setUp() throws Exception {
                       server.waitForStringInLog("CWWKF0008I"));
         assertNotNull("Server should report it has started",
                       server.waitForStringInLog("CWWKF0011I"));
+        assertNull("Server should have not reported BootstrapContext not arriving", server.waitForStringInLog("BootstrapContext service event has not arrived"));
```

Comment:
Since we are expecting the string to _not_ be present, this will add a lot of time to the bucket in LITE mode because we wait for a few minutes on `waitForStringInLog` on LITE mode and about 2-3x that on FULL mode. Instead, since we are not expecting the string to be in logs, lets use a method like `verifyStringNotInLogUsingMark`